typescript/sdk/src/metadata/agentConfig.ts

```diff
@@ -92,6 +92,30 @@ export type AgentSignerCosmosKey = z.infer<typeof AgentSignerNodeSchema>;
 export type AgentSignerNode = z.infer<typeof AgentSignerNodeSchema>;
 export type AgentSigner = z.infer<typeof AgentSignerSchema>;
 
+// Additional chain metadata for Cosmos chains required by the agents.
+const AgentCosmosChainMetadataSchema = z.object({
```

Comment:
Should `buildAgentConfig` be updated to use this?

Reply:
Because for now buildAgentConfig is just used for EVM chains atm, just gonna make a comment about `buildAgentConfig` needing some work before being used in non-EVM contexts for now